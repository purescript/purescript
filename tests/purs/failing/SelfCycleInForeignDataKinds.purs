-- @shouldFailWith CycleInKindDeclaration
module Main where

foreign import data Foo :: Foo
