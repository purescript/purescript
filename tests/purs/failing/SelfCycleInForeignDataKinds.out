Error found:
at tests/purs/failing/SelfCycleInForeignDataKinds.purs:4:1 - 4:31 (line 4, column 1 - line 4, column 31)

  A kind declaration '[33mFoo[0m' may not refer to itself in its own signature.


See https://github.com/purescript/documentation/blob/master/errors/CycleInKindDeclaration.md for more information,
or to contribute content related to this error.

